frontend/result/script.js

```diff
@@ -4,19 +4,55 @@ let returnButton = document.querySelector(".main__return-button");
 let city = params.get("city");
 let url = document.location.origin + `/api/get/weather/${city}`;
 let resPage = document.querySelector(".res-page")
+
 returnButton.addEventListener("click", () => {
   window.location.replace(window.location.origin);
 });
+
+class Weather {
+  constructor(temperature, description, feels_like, country, humidity, wind_speed) {
+    this.temperature = temperature
+    this.descriprtion = description
+    this.feels_like = feels_like
+    this.country = country
+    this.humidity = humidity
+    this.wind_speed = wind_speed
+  }
+  match() {
```

Comment:
make the name of method more informative

Reply:
done